st2reactor/st2reactor/sensor/base.py

```diff
@@ -23,8 +23,10 @@ def __init__(self, sensor_service, config=None):
         :keyword config: Sensor config.
         :type config: ``dict`` or None
         """
-        self._sensor_service = sensor_service
-        self._config = config or {}
+        self._sensor_service = sensor_service  # Deprecate in the future
+        self.sensor_service = sensor_service
+        self._config = config or {}  # Deprecate in the future
+        self.config = self._config
 
```

Comment:
I originally missed this PR (I blame thunderbird sync!) and that's how I've done it utilizing @property - https://github.com/StackStorm/st2/pull/2698/files#diff-f120a0125bcd3ec86f1d85020ed530ffR26